pkg/operator/controller/externaldns/credentials-request_test.go

```diff
@@ -0,0 +1,287 @@
+package externaldnscontroller
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	cco "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
+	corev1 "k8s.io/api/core/v1"
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	operatorv1alpha1 "github.com/openshift/external-dns-operator/api/v1alpha1"
+	controller "github.com/openshift/external-dns-operator/pkg/operator/controller"
+)
+
+var (
+	testScheme = runtime.NewScheme()
+)
+
+func init() {
+	if err := clientgoscheme.AddToScheme(testScheme); err != nil {
+		panic(err)
+	}
+	if err := operatorv1alpha1.AddToScheme(testScheme); err != nil {
+		panic(err)
+	}
+
+	if err := cco.AddToScheme(testScheme); err != nil {
+		panic(err)
+	}
+}
```

Comment:
`test/` package (in this same directory) already has the test scheme initialized.

Reply:
I am getting this error after removing it

== RUN   TestEnsureCredentialsRequest
--- FAIL: TestEnsureCredentialsRequest (0.00s)
panic: failed to add runtime object &{{ } {test     999 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] []  []} {[] {AWS 0xc0000c05b0 <nil> <nil> <nil> <nil>} {{OpenShiftRoute map[] <nil> <nil> <nil> <nil>}  []} [public-zone]} {[] 0 []}} to fake client: no kind is registered for the type v1alpha1.ExternalDNS in scheme "github.com/openshift/external-dns-operator/pkg/operator/controller/externaldns/credentials-request_test.go:25" [recovered]
	panic: failed to add runtime object &{{ } {test     999 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] []  []} {[] {AWS 0xc0000c05b0 <nil> <nil> <nil> <nil>} {{OpenShiftRoute map[] <nil> <nil> <nil> <nil>}  []} [public-zone]} {[] 0 []}} to fake client: no kind is registered for the type v1alpha1.ExternalDNS in scheme "github.com/openshift/external-dns-operator/pkg/operator/controller/externaldns/credentials-request_test.go:25"

goroutine 48 [running]:
testing.tRunner.func1.2(0x17e5780, 0xc00063ba20)